src/components/inputs/localized-money-input/localized-money-input.js

```diff
@@ -0,0 +1,319 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import without from 'lodash.without';
+import oneLine from 'common-tags/lib/oneLine';
+import { injectIntl } from 'react-intl';
+import Spacings from '../../spacings';
+import Constraints from '../../constraints';
+import {
+  createLocalizedDataAttributes,
+  getHasErrorOnRemainingLanguages,
```

Comment:
did we decide where or not it makes sense to make these into their own functions, instead of reusing the language ones?

Reply:
Maybe renaming the function would be good enough?